docs/source/gbd2017_models/causes/ischemic_heart_disease/index.rst

```diff
@@ -211,13 +238,25 @@ State and Transition Data Tables
      - Mid-year population for given sex/age/year/location
      - 
    * - prevalence_s{sid}
-     - como
+     - dismod-mr
      - Prevalence of sequela with id {id}
      - 
    * - disability_weight_s{sid}
      - YLD appendix
      - Disability weight of sequela with id {id}
      - 
+   * - incidence_493
```

Comment:
If incidence is from dismod, don't we use meid? The id 493 is different from other ids. Which one's correct?

Reply:
I believe the source would be como, here!